### PR TITLE
Add the third part of the KT TTT Tutorial.

### PR DESCRIPTION
**Pull request by SHeimlich**

### CODE REVIEW

**Comment by SHeimlich** [comment]
NOTE: this cannot be pushed until Myk re-lands the Handles PR.

File: particles/Tutorial/Kotlin/Demo/src/pt3/TTTBoard.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    private val events = Collection(this, "events") { TTTBoard_Events() }
+    private var clicks = 0.0
+    private val emptyBoard = listOf("", "", "", "", "", "", "", "", "")
+
+    init {
+        eventHandler("onClick") { eventData ->
+            events.store(TTTBoard_Events(
+                    type = "move",
+                    move = eventData["value"]?.toDouble() ?: -1.0,
+                    time = clicks
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/src/pt3/TTTBoard.kt]: Indented too far

File: particles/Tutorial/Kotlin/Demo/src/pt3/TTTBoard.kt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        val boardList = gameState.get()?.board?.split(",") ?: emptyBoard
+        val boardModel = mutableListOf<Map<String, String?>>()
+        boardList.forEachIndexed { index, cell ->
+            boardModel.add(mapOf("cell" to cell, "value" to index.toString()))
+        }
+
+        return mapOf(
+                "buttons" to mapOf(
+                        "\$template" to "button",
+                        "models" to boardModel
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/src/pt3/TTTBoard.kt]: Here too

File: particles/Tutorial/Kotlin/Demo/src/pt3/TTTGame.kt [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+    private val winningSequences = arrayOf(
+            arrayOf(0, 1, 2),
+            arrayOf(3, 4, 5),
+            arrayOf(6, 7, 8),
+            arrayOf(0, 3, 6),
+            arrayOf(1, 4, 7),
+            arrayOf(2, 5, 8),
+            arrayOf(0, 4, 8),
+            arrayOf(2, 4, 6)
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/src/pt3/TTTGame.kt]: Here too.

Everywhere!

**Pull request merged**